components/signup/signup.scss

```diff
@@ -0,0 +1,387 @@
+    }
+
+    20%,
+    80% {
+        transform: translate3d(2px, 0, 0);
+    }
+
+    30%,
+    50%,
+    70% {
+        transform: translate3d(-4px, 0, 0);
+    }
+
+    40%,
+    60% {
+        transform: translate3d(4px, 0, 0);
+    }
+}
+
+@media screen and (min-width: 1680px) {
+    .signup-body .signup-body-content {
+        .signup-body-message,
+        .signup-body-card,
+        .signup-body-no-login {
+            width: 610px;
+        }
+
+        .signup-body-message .signup-body-message-title {
+            padding-right: 130px;
+            margin-top: 48px;
+        }
+    }
+}
+
+@media screen and (max-width: 1199px) {
+    .signup-body .signup-body-content {
+        flex-direction: column;
+
+        .signup-body-message,
+        .signup-body-card,
+        .signup-body-no-login {
+            width: 640px;
+        }
+
+        .signup-body-message {
+            align-self: center;
+            padding: 24px;
+
+            .signup-body-message-title {
+                padding-right: 0;
+                font-size: 64px;
+                line-height: 76px;
+            }
+
+            .signup-body-message-subtitle {
+                padding: 0;
+                margin: 0;
+            }
+
+            .signup-body-message-svg {
+                display: none;
+            }
+
+            &.custom-branding {
+                display: none;
+            }
+        }
+
+        .signup-body-card {
+            border: none;
+            margin: 0;
+            background-color: unset;
+            box-shadow: none;
+
+            .signup-body-card-content {
+                padding: 16px 24px;
+
+                .signup-body-card-title {
+                    display: none;
+                }
+
+                .signup-body-card-banner {
+                    margin: 0 0 32px;
+                }
+
+                .signup-body-card-form {
+                    .signup-body-card-form-email-input {
+                        margin-top: 0;
+                    }
+
+                    .signup-body-card-form-email-input,
+                    .signup-body-card-form-name-input,
+                    .signup-body-card-form-password-input {
+                        background-color: var(--center-channel-bg);
+                    }
+                }
+            }
+
+            &.custom-branding {
+                .signup-body-card-content {
+                    .signup-body-card-title {
+                        display: block;
+                        font-size: 32px;
+                        line-height: 40px;
+                    }
+
+                    .signup-body-custom-branding-markdown,
+                    .signup-body-message-subtitle {
+                        display: block;
+                        margin: 0 0 32px;
+                    }
+                }
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 699px) {
+    .signup-body .signup-body-content {
+        min-width: 414px;
```

Comment:
do we ned this? What about phones with widths less than this. The smallest phone could be 375px;

Reply:
Not really needed, can remove it and the layout will just keep shrinking:
<img width="474" alt="Screen Shot 2022-05-18 at 12 58 05 PM" src="https://user-images.githubusercontent.com/79058848/169111369-9f6fb62d-a80e-4369-a641-93ef0875067f.png">